modules/configuring-default-route-timeouts.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * knative-serving/config-applications/configuring-revision-timeouts.adoc
+:_content-type: PROCEDURE
+[id="configuring-default-route-timeouts_{context}"]
+= Configuring the default route timeouts per revision
+
+By configuring the route timeouts per revision, you can fine-tune timeouts for workloads with unique requirements, such as AI or data processing applications, without impacting the global timeout settings for other services.
+You can configure the timeouts of a specific revision by updating your `KnativeServing` custom resource (CR), the Service definition, and using the `serving.knative.openshift.io/setRouteTimeout` annotation to adjust the OCP route timeout.
```

Comment:
```suggestion
You can configure the timeouts of a specific revision by updating your `KnativeServing` custom resource (CR), the `Service` definition, and using the `serving.knative.openshift.io/setRouteTimeout` annotation to adjust the OpenShift route timeout.
```

Should it be "OpenShift route timeout."? We don't use OCP though, you can use the usual openshift attribute that is used in serveless